### PR TITLE
Changed fisher_data path and `rm` command

### PR DESCRIPTION
**Pull request by jsmitley**
Moved fisher_data path from .config to .local/share to match latest fisher release
Changed `rm` to `command rm` so it doesn't try to use a fish `rm` function

### CODE REVIEW

**Comment by reitzig** [comment]
We should probably use XDG variables like fisher. :thinking: 

Will look at this when I next touch the project. Won't be tomorrow, sorry.

**Comment by jsmitley** [comment]
Good point, I made that change now. Please look when you have a chance. Thanks!

File: conf.d/sdk.fish [hunk 1]
```diff
@@ -44,7 +47,7 @@ function __fish_sdkman_run_in_bash
              env | grep -i -E \"^(`echo \${SDKMAN_CANDIDATES_CSV} | sed 's/,/|/g'`)_HOME\" >> $pipe;
              echo \"SDKMAN_OFFLINE_MODE=\${SDKMAN_OFFLINE_MODE}\" >> $pipe;
              echo \"SDKMAN_ENV=\${SDKMAN_ENV}\" >> $pipe" # it's not an environment variable!
-    set bashDump (cat $pipe; rm $pipe)
+    set bashDump (cat $pipe; command rm $pipe)
```
Comment (reitzig) [inline conf.d/sdk.fish:50]: Can you please explain which problem this change solves? :thinking:

Comment (jsmitley) [reply conf.d/sdk.fish:50]: Sorry I'm late with this! I have a separate command in my config that overrides `rm` (calls trash instead), which breaks this call. If I use the `command` to call `rm`, it seems to get the same original effect.

Comment (jsmitley) [reply conf.d/sdk.fish:50]: @reitzig Any thoughts on this? I don't know of a more fish-y way to handle this particular command override 🤷

Comment (reitzig) [reply conf.d/sdk.fish]: I don't think it's feasible to look out for overrides of "basic" commands. :/ See also #23.

**Comment by reitzig** [comment]
Sorry for the long silence.

> Moved fisher_data path from .config to .local/share to match latest fisher release

Somehow, only a README change is left on this PR. I can't seem to remember what this part was about. Is there still an issue?

**Pull request closed without merging**